src/v2/Components/ArtworkFilter/__tests__/ArtworkFilterMobileActionSheet.jest.tsx

```diff
@@ -10,10 +11,20 @@ import { ArtworkFilters } from "../ArtworkFilters"
 jest.mock("v2/Utils/Hooks/useMatchMedia", () => ({
   __internal__useMatchMedia: () => ({ sm: true }),
 }))
+jest.mock("v2/System/Analytics/useTracking")
```

Comment:
mocks `useTracking` since the tests were failing without this (after the addition of `useTrackVariantView` hook in `ArtworkFilters/index.ts`

Reply:
(Note this is a common pattern in our tests where tracking intersects)